migrator/migration.py

```diff
@@ -13,24 +13,43 @@
     migration_plan_instance_name,
     domain_with_cdn_plan_guid,
 )
-from migrator.models import CdnRoute
+from migrator.models import CdnRoute, DomainRoute
 from migrator.smtp import send_email
 
 logger = logging.getLogger(__name__)
 
 
 def find_active_instances(session):
-    query = session.query(CdnRoute).filter(CdnRoute.state == "provisioned")
-    routes = query.all()
+    cdn_routes = find_active_cdn_instances(session)
+    domain_routes = find_active_domain_instances(session)
+    routes = [*cdn_routes, *domain_routes]
```

Comment:
I initially had a question about the combination of these two route types here and if they'd be processed the same, but then I saw the factory below, so 👍 - nicely done!